container/controller.go

```diff
@@ -287,6 +292,17 @@ func NewController(options ControllerOptions) (*Controller, error) {
 		errc <- nil
 	}()
 
+	// Load the delegate keys and auth tokens first so there's no race btwn starting the watcher routines
```

Comment:
Yeah, I guess this is fine now. Before, services would be able to be scheduled to a host before it had authenticated, so the watching was necessary.